projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -268,7 +268,19 @@ public function add_upgrades_menu( $plan = null ) {
 	public function add_options_menu() {
 		parent::add_options_menu();
 
-		add_submenu_page( 'options-general.php', esc_attr__( 'Security', 'jetpack' ), __( 'Security', 'jetpack' ), 'manage_options', 'https://wordpress.com/settings/security/' . $this->domain, null, 2 );
+		$show_security_nav_item = Jetpack_Plan::supports( 'security-settings' );
+
+		if ( $show_security_nav_item ) {
```

Comment:
Just a minor one. Do we really need to instance a var? Please feel free to ignore this comment:


```suggestion
		if ( Jetpack_Plan::supports( 'security-settings' ) ) {
```

Reply:
Yeah probably not, thanks for pointing that out. Made the change here: https://github.com/Automattic/jetpack/pull/20384/commits/e6302d10b93384f5070806e6df4dc7add9430fbb